x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -324,11 +324,7 @@ export const RulesList = ({
 
   useEffect(() => {
     loadData();
-  }, [loadData, refresh]);
-
-  useEffect(() => {
-    loadData();
-  }, [loadData, percentileOptions]);
+  }, [loadData, refresh, percentileOptions]);
```

Comment:
this was doing 2 api calls

Reply:
I think that might create a bug when we changed the percentile in the column. I think we can do it smarter.